New languages, new string, new flags, readme description and improving definition in some language files

JhowDroid opened the pull request:
- adding new language file: se.rb (Swedish Translation - Svenska
  Översättning);
- adding new string "about_icons" to Deutch and Spanish language files;
- Improving definition to the string "about_icons" in language files:
  en.rb, pt.rb, pt-PT.rb;

@davidsantos-br

JhowDroid committed:
Merge remote-tracking branch 'refs/remotes/davidsantos-br/master' into language

JhowDroid committed:
Swedish Translation - Svenska Översättning

* adding new language file: se.rb (Swedish Translation - Svenska
Översättning)

JhowDroid committed:
Adding new string "about_icons"

* adding new string "about_icons" to Deutch and Spanish language files;

JhowDroid committed:
Improving definition of the string "about_icons" in some language files

* Improving definition of the string "about_icons" in language files:
en.rb, pt.rb, pt-PT.rb

davidsantos-br commented:
Você usou uma versão sem as últimas alterações nos arquivos de e es. No inglês, fica melhor “Country flag icons”.

JhowDroid committed:
Merge remote-tracking branch 'refs/remotes/davidsantos-br/master'

JhowDroid committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into language

JhowDroid commented:
Concordo e já identifiquei a diferença. Em instantes subirei a correção

JhowDroid committed:
Correcting file version and improving definition

* Correcting file version for: de.rb, es.rb
* Improving definition to the string "about_icons" in language files:
en.rb as suggested by @davidsantos-br

JhowDroid committed:
Improving definition to the string "about_icons" for some files

Improving definition to the string "about_icons" in language files:
de.rb and se.rb

JhowDroid committed:
Adding a country flag

Adding country flag for swedish language

JhowDroid commented:
Finalizado. Agora está completo.
@davidsantos-br

JhowDroid committed:
Editing readme file inserting swedish language

Adding swedish language description to readme file.

JhowDroid committed:
French Translation - Traduction Française

* Adding new language file: fr.rb (French Translation - Traduction
Française)
* Adding new country flag for french language

JhowDroid committed:
Adding French language to README files

* Adding French language to README files;
* Reordering the languages in the README file;
